examples/tv-app/tv-common/src/AppTv.cpp

```diff
@@ -627,6 +633,31 @@ bool ContentAppFactoryImpl::UninstallContentApp(uint16_t vendorId, uint16_t prod
     return false;
 }
 
+void ContentAppFactoryImpl::SetAppInstallationStatus(uint16_t vendorId, uint16_t productId, CommissionerDeclaration::CdError status)
+{
+    std::map<uint16_t, Protocols::UserDirectedCommissioning::CommissionerDeclaration::CdError>::iterator it;
+    it = mAppInstallationStatus.find(vendorId);
+    if (it != mAppInstallationStatus.end())
+    {
+        mAppInstallationStatus[vendorId] = status;
```

Comment:
in theory, a single vid could have multiple apps and so it seems like we should index this on vendorId and productId

Reply:
Created a search index <pid>:<vid>